### PR TITLE
Update handbook for octal and binary literal

### PR DESCRIPTION
**Pull request by yuit**
Issue #66

### CODE REVIEW

File: pages/Basic Types.md [hunk 1]
```diff
@@ -14,10 +14,17 @@ var isDone: boolean = false;
 # Number
 
 As in JavaScript, all numbers in TypeScript are floating point values.
-These floating point numbers get the type `number`.
+These floating point numbers get the type `number`. In Addition to hexadecimal and decimal literal, TypeScript also
```
Comment (sandersn) [inline pages/Basic Types.md]: 1. Lowercase 'Addition'
2. Pluralize 'literal'
3. Put the new sentence on its own line.

File: pages/Basic Types.md [hunk 2]
```diff
@@ -14,10 +14,17 @@ var isDone: boolean = false;
...
+supports binary and octal literal introduced in ECMAScript 2015.
```
Comment (sandersn) [inline pages/Basic Types.md]: literal should also be plural here.

File: pages/Basic Types.md [hunk 3]
```diff
@@ -14,10 +14,17 @@ var isDone: boolean = false;
...
 
 ```TypeScript
 var height: number = 6;
+var hexLiteral1 = 0x9837abdef;
+var hexLiteral2= 0X9837ABDEf;
```
Comment (sandersn) [inline pages/Basic Types.md]: space before =

**Comment by sandersn** [comment]
Some minor formatting comments. Otherwise, :+1:

File: pages/Basic Types.md [hunk 4]
```diff
@@ -14,10 +14,17 @@ var isDone: boolean = false;
 # Number
 
 As in JavaScript, all numbers in TypeScript are floating point values.
-These floating point numbers get the type `number`.
+These floating point numbers get the type `number`. In Addition to hexadecimal and decimal literal, TypeScript also
+supports binary and octal literal introduced in ECMAScript 2015.
 
 ```TypeScript
 var height: number = 6;
```
Comment (DanielRosenwasser) [inline pages/Basic Types.md]: dec

File: pages/Basic Types.md (hunk 1 above)
Comment (yuit) [inline pages/Basic Types.md]: Thanks!

**Comment by mhegazy** [comment]
:+1:

File: pages/Basic Types.md [hunk 5]
```diff
@@ -15,9 +15,13 @@ var isDone: boolean = false;
...
 As in JavaScript, all numbers in TypeScript are floating point values.
 These floating point numbers get the type `number`.
+In addition to hexadecimal and decimal literals, TypeScript also supports binary and octal literals introduced in ECMAScript 2015.
 
 ```TypeScript
-var height: number = 6;
+var decLiteral = 6;
+var hexLiteral = 0x9837abdef;
+var binaryLiteral = 0b0010;
+var octalLiteral = 0o74563;
```
Comment (DanielRosenwasser) [inline pages/Basic Types.md]: These all should get a type annotation like in the other examples.

Comment (yuit) [inline pages/Basic Types.md]: Sure. Though it is already listed under number type though X)

**Pull request merged**